genon/preprocessor/facade/convert_processor.py

```
@@ -3039,48 +3040,65 @@ def extract_ocr_fields(resp: dict):
-                    rec_texts, rec_scores, rec_boxes = extract_ocr_fields(result)
+                        # docling 셀 bbox(BOTTOMLEFT) → 페이지 이미지 픽셀 좌표(TOPLEFT)
+                        crop = (
+                            cell.bbox
+                            .to_top_left_origin(page_height=page.size.height)
+                            .scale_to_size(old_size=page.size, new_size=page.image.size)
+                        )
+                        x0, y0, x1, y1 = crop.as_tuple()
+                        # 정규화 + 페이지 경계 클램프 + degenerate skip
+                        x0, x1 = sorted((x0, x1))
+                        y0, y1 = sorted((y0, y1))
+                        x0 = max(0, min(x0, W)); x1 = max(0, min(x1, W))
```

Ruff (0.15.20)

[error] Multiple statements on one line (semicolon)

(E702)